Sources/AsyncServer.swift

```diff
@@ -1,9 +1,9 @@
 public protocol AsyncServer {
-    func serve(responder: AsyncResponder, on host: String, at port: Int) throws
+    func serve(_ responder: AsyncResponder, on host: String, at port: Int) throws
```

Comment:
I think `Server().serve(app, at: 8080)` is too much, 
i believe it's much better to stick with the standard and familiar api here. https://github.com/open-swift/S4/pull/50